tests/unit/s2n_ktls_test_utils_test.c

```diff
@@ -98,8 +105,10 @@ int main(int argc, char **argv)
             EXPECT_OK(s2n_test_init_ktls_io_stuffer(server, client, &io_pair));
 
             struct iovec send_msg_iov = { .iov_base = test_data, .iov_len = S2N_TEST_TO_SEND };
-            struct msghdr send_msg = { .msg_iov = &send_msg_iov, .msg_iovlen = 1, .msg_control = &test_record_type };
-            /* sendmsg */
+            struct msghdr send_msg = { .msg_iov = &send_msg_iov, .msg_iovlen = 1 };
+            char control_buf[S2N_CONTROL_BUF_SIZE] = { 0 };
+            EXPECT_OK(s2n_ktls_set_control_data(&send_msg, control_buf, sizeof(control_buf),
```

Comment:
Why are we retrospectively adding these functions to these tests? How does it change the test?